app/code/Magento/Catalog/Block/Breadcrumbs.php

```diff
@@ -4,6 +4,8 @@
  * See COPYING.txt for license details.
  */
 
+// @codingStandardsIgnoreFile
```

Comment:
Please, don't use this annotation. Even if this page contains some legacy approaches, they should be visible. 
On the other hand, I see that some of your edits contain changes that don't meet the PSR coding standards (line length exceeds 120 chars). Please, remove this annotation and run the static tests once again. 

Thank you

Reply:
I've done this now, and fixed the PSR coding standards issue you referenced.

As of now it is passing all tests bar the Functional Tests. What needs to be done next?

Reply:
By adding this directive you don't "fix" the coding standard issues but tell the system to ignore them :) 
Using such directives is not allowed in most cases (including this one). If there are failing static tests, every issue that causes the failing test should be addressed by adjusting the code for meeting the standards.

Thank you.